javascriptv3/example_code/glue/tests/clean-up-databases.unit.test.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe, expect, it, vi } from "vitest";
+import { makeCleanUpDatabasesStep } from "../scenarios/basic/steps/clean-up-databases.js";
+import { mockPrompter } from "./mock-prompter.js";
+
+describe("clean-up-databases", () => {
+  it("should not ask, or attempt, to delete, any databases if none are found", () => {
+    const getDatabases = vi.fn(async () => ({ DatabaseList: [] }));
```

Comment:
Can we try mocking the entire @aws-sdk/glue GlueClient? Something like

```js
import glue from '@aws-sdk/glue';
import step from '../steps/step';

vi.mock(glue);

test('Step Test', () => {
  // const listJobs = vi.fn(async () => ({ JobNames: [] }));

  vi.mocked(glue.GlueClient.prototype.send).andResolve({ JobNames: []});
});
```

Reply:
See `clean-up-crawler.unit.test.js`